mne/preprocessing/annotate_nan.py

```diff
@@ -0,0 +1,47 @@
+# Author: David Julien <david.julien@ifsttar.fr>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+
+import warnings
+
+from ..annotations import Annotations
+from ..utils import _mask_to_onsets_offsets
+
+
+def annotate_nan(raw):
```

Comment:
And add the `%(verbose)s` in the parameters
```suggestion
@verbose
def annotate_nan(raw, *, verbose=None):
```
It's always good to give a `verbose` arg to things that can potentially warn so that you can easily disable the warnings